tokio/src/util/mod.rs

```diff
@@ -10,6 +10,9 @@ pub(crate) mod metric_atomics;
 #[cfg(any(feature = "rt", feature = "signal", feature = "process"))]
 pub(crate) mod once_cell;
 
+#[cfg(feature = "net")]
+pub(crate) mod blocking_check;
```

Comment:
Consider re-exporting `check_socket_for_blocking` here to make the imports more nice.